src/ccmain/tessedit.cpp

```diff
@@ -308,7 +308,8 @@ int Tesseract::init_tesseract(const std::string &arg0, const std::string &textba
   // Load the rest into sub_langs_.
   // A range based for loop does not work here because langs_to_load
   // might be changed in the loop when a new submodel is found.
-  for (auto &lang_to_load : langs_to_load) {
```

Comment:
I should have read my own comment which said that using a range based loop is a bad idea here.

Reply:
LOL, but seriously, maybe you should add `WARNING:` in the start of the comment line.

Reply:
Good idea. Done now.